pkgs/misc/tmux-plugins/default.nix

```
@@ -216,19 +216,19 @@ in rec {
 
   extrakto = mkTmuxPlugin {
     pluginName = "extrakto";
-    version = "unstable-2021-04-04";
+    version = "unstable-2023-10-21";
```

```suggestion
    version = "unstable-2023-10-05";
```
This should always be the date of the commit.